src/marqo/s2_inference/clip_utils.py

```diff
@@ -116,7 +116,13 @@ def load_image_from_path(image_path: str, image_download_headers: dict, timeout_
             if metrics_obj is not None:
                 metrics_obj.stop(f"image_download.{image_path}")
     else:
-        raise UnidentifiedImageError(f"input str of `{image_path}` is not a local file or a valid url")
+        raise UnidentifiedImageError(f"Input str of {image_path} is not a local file or a valid url. "
+                                     f"If you are using Marqo Cloud, please note that images can only be downloaded "
+                                     f"from a URL and local files are not supported. "
+                                     f"If you are running Marqo in a Docker container, you will need to use a Docker "
+                                     f"volume so that your container can access host files. "
+                                     f"For more information, please refer to: "
+                                     f"https://docs.marqo.ai/latest/Guides/Advanced-Usage/images/")
```

Comment:
Use `marqo_docs` module so link it tested and has the right version

Reply:
addressed